gpflow/base.py

```diff
@@ -307,6 +304,19 @@ def _run_op(a, *args, **kwargs):
 Parameter._OverloadAllOperators()
 tf.register_tensor_conversion_function(Parameter, lambda x, *args, **kwds: x.read_value())
 
+TensorLike = Union[np.ndarray, tf.Tensor, tf.Variable, Parameter]
```

Comment:
```suggestion

TensorLike = Union[np.ndarray, tf.Tensor, tf.Variable, Parameter]
```
(to visually set it more apart from the Parameter class details; black is happy with two-line blanks in this context)

Reply:
will do